src/test/ruby/test_cipher.rb

```diff
@@ -475,4 +475,19 @@ def test_encrypt_aes_cfb_20_incompatibility
     end
   end
 
+  def test_encrypt_aes_256_cbc_modifies_buffer
+    cipher = OpenSSL::Cipher.new("AES-256-CBC")
+    cipher.key = "a" * 32
+    cipher.encrypt
+    buffer = ''
+    actual = cipher.update('bar' * 10, buffer)
+    if jruby?
+      expected = "\xE6\xD3Y\fc\xEE\xBA\xB2*\x0Fr\xD1\xC2b\x03\xD0"
```

Comment:
maybe a bug?

Reply:
@ahorek I just saw this.  I don't know.  I would expect the two impls to generate the same data.